_pages/cfp.md

```diff
@@ -1,22 +1,79 @@
+advanced knowledge dissemination. This allows the field of Information Security to grow 
+in breadth and depth, and continue in its pursuit of highly advanced, scientifically 
+based knowledge.
+
+We are a practitioner-focused event that attracts a wide range of attendees ranging 
+from engineers & security leaders to executive decision makers. This is a great 
+opportunity for you to showcase your work and share your ideas with industry peers. 
+Bonus points if you can tie your presentation to this year’s conference theme!
+
+The BSides community has continuously raised the bar and put the info back in 
+infosec. We thank each and every member/participant/organizer of this community for 
+their hard work, sweat, and relentless pursuit of making high quality content accessible.
+We celebrate diverse approaches and points of view and believe that our commitment to 
+diversity fosters innovation. BSidesSF is continuing to build upon a culture that drives 
+our community to include a wide range of thoughts and ideas.
+
+# **When/Where**
+
+BSidesSF will take place on April 26-27, 2025. The conference will be located 
+at City View at the Metreon in downtown San Francisco.
+
+# **Theme**
+
+This year, we dive into uncharted territory of security under the theme "Here Be 
+Dragons." Like the ancient maps that marked unknown regions with dragons, today's 
+landscape is filled with threats that are complex, evolving, and often unpredictable. 
+As security practitioners, we are the navigators of this vast, sometimes dangerous 
+terrain. We're opening the call to participate in our 2025 conference - come share your 
+experiences exploring new challenges, discovering the unknown, battling cutting-edge 
+technologies, and sharing strategies to confront the 'dragons' lurking in the shadows. ⚔️
+You can't spell dystopia without AI
+
+# **Submit to the CFP!**
 
-We're always looking for folks to help us shape BSidesSF content! CFP reviewers typically spend 30-40 hours (if not more) around the Dec 2024/Jan 2025 timeframe. We also typically ask reviewers for recommendations for new keynote speakers. You will receive free admission to BSidesSF at the end for all your hard work.
+Please go **---> [HERE](https://pretalx.com/bsidessf2025/cfp) <---** for submissions for 
+Presentations, Panels, Workshops, and Birds of a Feather
 
-Submit an application here: [https://forms.gle/4BGiKQC1Cdko4cYdA](https://forms.gle/4BGiKQC1Cdko4cYdA)
+# **Propose A Village**
 
-# **Questions?**
+Village proposals will be accepted soon!
 
+## **Questions?**
 E-mail program [at] bsidessf.org
 
-Or ask us in our Slack! (use the Slack logo at top right of page for an invite to the Slack)
+You can also ping **@program-ops** in 
+**#[2025-cfp-helpdesk](https://bsidessf.slack.com/archives/C07J90PD98D)** on the 
+BSidesSF Slack (use the Slack logo at top right of page for an invite to the Slack).
+
+
+# **Interested in participating on the BSidesSF 2025 Program Committee?**
```

Comment:
Is this still needed?

Reply:
@jknecht2 is the authority here

Reply:
We can get rid of it - we said apps closed 9/18 for this year